### PR TITLE
SENTRY_DIST accepts non-number values on Android

### PR DESCRIPTION
**Pull request by marandaneto**
## :loudspeaker: Type of change

- [X] Bugfix
- [ ] New feature
- [ ] Enhancement
- [ ] Refactoring

## :bulb: Motivation and Context
https://github.com/getsentry/sentry-react-native/discussions/2394

## :green_heart: How did you test it?
Running

## :pencil: Checklist

- [X] I reviewed submitted code
- [ ] I added tests to verify changes
- [X] All tests passing
- [X] No breaking changes

### CODE REVIEW

File: sample/yarn.lock [hunk 1]
```diff
@@ -4875,9 +4875,9 @@ camelcase@^6.0.0:
   integrity sha512-c7wVvbw3f37nuobQNtgsgG9POC9qMbNuMQmTCqZv23b6MIz0fcYpBiOlv9gEN/hdLdnZTDQhg6e9Dq5M1vKvfg==
 
 caniuse-lite@^1.0.30001265:
-  version "1.0.30001267"
-  resolved "https://registry.yarnpkg.com/caniuse-lite/-/caniuse-lite-1.0.30001267.tgz#b1cf2937175afc0570e4615fc2d2f9069fa0ed30"
-  integrity sha512-r1mjTzAuJ9W8cPBGbbus8E0SKcUP7gn03R14Wk8FlAlqhH9hroy9nLqmpuXlfKEw/oILW+FGz47ipXV2O7x8lg==
+  version "1.0.30001370"
+  resolved "https://registry.npmjs.org/caniuse-lite/-/caniuse-lite-1.0.30001370.tgz"
+  integrity sha512-3PDmaP56wz/qz7G508xzjx8C+MC2qEm4SYhSEzC9IBROo+dGXFWRuaXkWti0A9tuI00g+toiriVqxtWMgl350g==
```
Comment (marandaneto) [inline sample/yarn.lock:4880]: transform[stderr]: Browserslist: caniuse-lite is outdated. Please run:
transform[stderr]:   npx browserslist@latest --update-db
transform[stderr]:   Why you should do it regularly: https://github.com/browserslist/browserslist#browsers-data-updating

File: sentry.gradle [hunk 2]
```diff
@@ -68,18 +68,17 @@ gradle.projectsEvaluated {
 
           try {
             if (versionCode instanceof String) {
-                versionCode = Integer.parseInt(versionCode);
+                versionCode = Integer.parseInt(versionCode)
+                versionCode = Math.abs(versionCode)
```
Comment (bitsandfoxes) [inline sentry.gradle:72]: Abs? The version code might end up being parsed as a negative value? I.e. `-3.14`

**Review by bitsandfoxes — APPROVED** [review]

File: sentry.gradle (hunk 2 above)
Comment (marandaneto) [reply sentry.gradle:72]: `abs` is already used, just moved the code.
`abs` make negative numbers into positive numbers, not the other way around.

Comment (bitsandfoxes) [reply sentry.gradle:72]: I got that. I was just surprised that the `versionCode` might end up with a negative value after being parsed from string.

**Pull request merged**